plugin/adsk/scripts/mayaUsd_preferenceTab.mel

```diff
@@ -46,8 +46,8 @@ global proc mayaUsd_PrefFileFormatChanged()
 	}
 
 	// Update the clipboard file format to match the format here.
-
-	if (`optionVar -exists $saveLayerFormatArgBinaryOptionPref`) {
+	int $haveClipboardSupport = `python("import mayaUsd.ufe as mufe; callable(getattr(mufe, 'setClipboardFileFormat', None))")`;
+	if ($haveClipboardSupport && `optionVar -exists $saveLayerFormatArgBinaryOptionPref`) {
```

Comment:
This clipboard function won't be there for Maya 2024 (and earlier) so check if function exists before calling it.